core/src/main/java/org/testcontainers/containers/output/Slf4jLogConsumer.java

```diff
@@ -9,13 +9,19 @@
  */
 public class Slf4jLogConsumer implements Consumer<OutputFrame> {
     private final Logger logger;
+    private String prefix = "";
 
     public Slf4jLogConsumer(Logger logger) {
         this.logger = logger;
     }
 
+    public Slf4jLogConsumer setPrefix(String prefix) {
```

Comment:
Please could you rename this `withPrefix` to match the prevailing fluent builder pattern?